nerdlandbot/commands/pets.py

```diff
@@ -0,0 +1,309 @@
+    ):
+        lang, guild_data = await asyncio.gather(
+            culture(ctx),
+            get_guild_data(ctx.message.guild.id),
+        )
+        if category_name is None:
+            msg = translate("pet_category_need_name", lang)
+            return await ctx.send(msg)
+
+        if not guild_data.user_is_admin(ctx.author):
+            gif = translate("not_admin_gif", lang)
+            return await ctx.send(gif)
+        
+        if await guild_data.add_new_pet_category(category_name.lower()):
+            msg = translate("pet_category_succes", lang).format(category_name)
+        else:
+            msg = translate("pet_category_error", lang)
+        return await ctx.send(msg)
+
+        
+
+    @commands.command(
+        name="show_categories",
+        brief="pet_show_categories_brief",
+        help="pet_show_categories_help"  
+    )
+    @commands.guild_only()
+    async def show_categories(self, ctx: commands.Context):
+        lang, guild_data = await asyncio.gather(
+            culture(ctx),
+            get_guild_data(ctx.message.guild.id),
+        )
+        if not guild_data.pets_categories:
+            msg = translate("pet_no_categories", lang)
+            return await ctx.send(msg)
+
+        msg = translate("pet_categories", lang)
+        for category in guild_data.pets_categories:
+            msg += f"\n -"
+            msg += category.capitalize()
+
+        return await ctx.send(msg)
+
+    @commands.command(
+        name="remove_category",
+        help="pet_remove_category_help",
+        usage="pet_remove_category_usage",
+        brief="pet_remove_category_brief"
+    )
+    @commands.guild_only()
+    async def remove_category(self, ctx: commands.Context, category_name: str):
+        lang, guild_data = await asyncio.gather(
+            culture(ctx),
+            get_guild_data(ctx.message.guild.id),
+        )
+        pets = guild_data.pets
+        category_name = category_name.lower()
+
+        for pet_id in pets.keys():
+            if pets[pet_id]["category"] == category_name:
+                msg = translate("pet_category_still_in_use", lang).format(pet_id)
+                return await ctx.send(msg)
+
+        if category_name is None:
+            translation_key = "pet_category_need_name"
+        else:
+            if await guild_data.remove_pet_category(category_name):
+                translation_key = "pet_category_removal_succes"
+
+        msg = translate(translation_key, lang)
+        return await ctx.send(msg)
+
+
+async def resize_and_save(ctx, attachment, pet_id):
+    guild_path = Path.cwd() / PETS_DIR_NAME / str(ctx.guild.id)
+    guild_path.mkdir(exist_ok=True, parents=True)
+
+    old_path = guild_path / f"{pet_id}_old.jpg"
+    new_path = guild_path / f"{pet_id}.jpg"
+
+    await attachment.save(str(old_path))
+    limit_img_size(old_path, new_path, IMAGE_SIZE, tolerance=5)
+    old_path.unlink()
+
+
+def check(author):
```

Comment:
Method name is a bit ambiguous, could do with a different name and perhaps a docstring?